### ISSUE TITLE
Inconsistent dll names in mpfr 4.2.1 builds for win64

### ISSUE THREAD

**Issue by 7Z0t99** [conda-forge/mpfr-feedstock#49]
### Comment:

Recent mpfr 4.2.1 builds for win64 like hbc20e70_3 distribute mpfr as libmpfr-6.dll, while a bit older builds like h64bf75a_1 distribute mpfr.dll and mpfr-6.dll. This change in dll names causes issues for other packages, see for example https://github.com/conda-forge/scipoptsuite-feedstock/issues/74
Is this change intentional?

**Comment by isuruf** [comment]
Yes, it is. The older packages were built with `mpir` while the new packages are built with `gmp`. So, any downstream package which had `mpfr` and `mpir` will get the older `mpfr` builds and when switching to `mpfr` and `gmp` you will get newer builds.

The problem is that you didn't have `mpir` as a dependency even though you were linking to `mpir`.
